packages/core/package.json

```diff
@@ -99,18 +99,18 @@
   "dependencies": {
     "@babel/polyfill": "^7.4.4",
     "@binary-com/binary-document-uploader": "^2.4.4",
+    "@deriv/bot-web-ui": "^1.0.0",
+    "@deriv/components": "^1.0.0",
     "@deriv/deriv-api": "^1.0.1",
-    "@deriv/p2p": "^0.2.0",
+    "@deriv/p2p": "^0.1.1",
```

Comment:
Why is the newer version for `p2p` overridden?

Reply:
ah probably kept the old version resolving the conflicts, i'll fix it